pandas/core/groupby/generic.py

```diff
@@ -132,7 +142,7 @@ def pinner(cls):
 class SeriesGroupBy(GroupBy):
     _apply_whitelist = base.series_apply_whitelist
 
-    def _iterate_slices(self):
+    def _iterate_slices(self) -> Iterator[Tuple[Hashable, Series]]:
```

Comment:
for return types, I _think_ best to use Iterable. the consumer doesn't much care how an iterable is created.

Reply:
Makes sense - updated per suggestion